sf-docker/sf-observability/configuration/grafana/datasources/datasources.yml

```diff
@@ -0,0 +1,57 @@
+﻿apiVersion: 1
+
+datasources:
+  - name: Loki
+    type: loki
+    uid: 'provisioned-loki'
+    access: proxy
+    url: http://loki:3100
+    version: 1
+    editable: false
+    isDefault: true
+
+  - name: Jaeger
+    type: jaeger
+    uid: 'provisioned-jaeger'
+    url: http://jaeger:16686/
+    jsonData:
+      tracesToLogs:
+        datasourceUid: 'provisioned-loki'
+        tags: ['job', 'instance', 'pod', 'namespace']
+        mappedTags:
+          - key: 'service.name'
+            value: 'service'
+        mapTagNamesEnabled: false
+        spanStartTimeShift: '1h'
+        spanEndTimeShift: '1h'
+        filterByTraceID: false
+        filterBySpanID: false
+      #tracesToMetrics:
+      #  datasourceUid: 'prom'
+      #  tags:
+      #    - key: 'service.name'
+      #      value: 'service'
+      #    - key: 'job'
+      #  queries:
+      #    - name: 'Sample query'
+      #      query: 'sum(rate(tempo_spanmetrics_latency_bucket{$__tags}[5m]))'
+    #secureJsonData:
+    #  basicAuthPassword: abcdefgh
+  
+  - name: Prometheus
+    type: prometheus
+    uid: 'provisioned-prometheus'
+    access: proxy
+    url: http://prometheus:9090
+    editable: false
+  
+  - name: PostgreSQL DB
+    uid: 'provisioned-pgsql'
+    type: postgres
+    url: pgsql:5432
+    database: smartface
+    user: postgres
+    secureJsonData:
+      password: 'Test1234'
+    jsonData:
+      sslmode: 'disable'
```

Comment:
Does any dashboard use this source?
Even if, it's not good to have it here with default password, which should be different on production installs.
Please remove this data source.

Reply:
removed